models.py

```diff
@@ -74,7 +74,7 @@ class Jurisdiction(db.Model):
         "AuditBoard", backref="jurisdiction", passive_deletes=True
     )
     contests = relationship(
-        "TargetedContestJurisdiction", backref="jurisdiction", passive_deletes=True
+        "ContestJurisdiction", backref="jurisdiction", passive_deletes=True
```

Comment:
Fwiw, I don’t think we’re ever gonna have a jurisdiction that is not targeted, for the record (I.e. one that only has opportunistic auditing, since opportunistic auditing is done only on ballots that have a targeted contest)

Reply:
I think that the `TargetedContestJurisdiction` table was just an association table to link `TargetedContest`s with `Jurisdiction`s, less a commentary on whether or not the jurisdiction is targeted. If that understanding is correct, then I think renaming both does makes sense.